codegen/src/test/resources/solidity/abifuncs/AbiFuncs.sol

```diff
@@ -0,0 +1,10 @@
+pragma solidity >= 0.8.7;
+
+contract AbiFuncs {
+
+
+    function hi(uint256 x) public {
```

Comment:
This seems like an empty solidity file, can't we reuse any pre-existing .sol file for the test?

Reply:
We can, but we can only test successful compilation if I use existing files, but cannot test java file output, if I modify the existing java file to add abiFuncs, we lose the test case of abiFuncs = false

Reply:
I also want to test java file output matches, if I use any existing file, I will have to modify those java files to add getABI* calls, but that will actually invalidate the test case with abiFunc=false

Reply:
Okay got it